subsys/logging/log_backend_rpc_history_ram.c

```diff
@@ -28,23 +28,32 @@ void log_rpc_history_init(void)
 
 void log_rpc_history_push(const union log_msg_generic *msg)
 {
-	uint32_t wlen;
-	union log_msg_generic *copy;
-	int len;
+	size_t wlen;
+	union mpsc_pbuf_generic *dst;
+	uint8_t *dst_data;
+	uint8_t *src_data;
+
+	wlen = log_msg_generic_get_wlen(msg);
+	dst = mpsc_pbuf_alloc(&log_history_pbuf, wlen, K_NO_WAIT);
+	if (!dst) {
+		/* No space to store the log */
+		return false;
```

Comment:
Does this compile? The function is `void`.

Reply:
Of course it does not, I pushed the update without ammending changes